api-distribution/src/test/java/com/synopsys/integration/alert/api/distribution/mock/MockAuditFailedEntryRepository.java

```diff
@@ -0,0 +1,14 @@
+package com.synopsys.integration.alert.api.distribution.mock;
+
+import java.util.UUID;
+import java.util.function.Function;
+
+import com.synopsys.integration.alert.database.audit.AuditFailedEntity;
+import com.synopsys.integration.alert.database.audit.AuditFailedEntryRepository;
+import com.synopsys.integration.alert.test.common.database.MockRepositoryContainer;
+
+public class MockAuditFailedEntryRepository extends MockRepositoryContainer<UUID, AuditFailedEntity> implements AuditFailedEntryRepository {
```

Comment:
This class is duplicated in another sub-project above.  We don't currently have a way to share mocked repositories to be used in testing.  For now this is duplicated.